Pipfile.lock

```diff
@@ -62,6 +62,14 @@
             "index": "pypi",
             "version": "==3.9.2"
         },
+        "gunicorn": {
```

Comment:
Gunicorn is added to the lock file and fixes the deployment error so this looks good to me.